src/main/java/org/jabref/gui/texparser/ParseTexResultViewModel.java

```diff
@@ -70,7 +73,7 @@ public void activeReferenceChanged(ReferenceViewModel reference) {
      * Search and import unknown references from associated BIB files.
      */
     public void importButtonClicked() {
-        ImportEntriesDialog dialog = new ImportEntriesDialog(databaseContext, BackgroundTask.wrap(() ->
+        ImportEntriesDialog dialog = new ImportEntriesDialog(databaseContext, BackgroundTask.wrap((Callable<List<BibEntry>>) () ->
```

Comment:
Why is it now necessary to add all these `Callable` statements. This is really ugly.

Reply:
That's because of the `Supplier` method. Both the `Supplier` and `Callable` are functional interfaces, which require explicit type casting now. But I do agree, it's kind of ugly.